### ISSUE TITLE
Failure with dev glue

### ISSUE THREAD

**Issue by jennybc** [rubenarslan/codebook#77]
This issue is inspired by doing revdep checks for glue. I'm going to temporarily back off on the associated change in glue, just so I can release without any breakage of other packages.

But please do consider this a heads up that, in the future, `glue::glue()` will error when `.envir` is not an actual environment. `.envir` has always been documented to be an environment and I'd like to make that actually true.

OTOH `glue_data()` *does* officially accept something "list-ish" as `.x`. So chances are you should switch to `glue_data(.x = list_ish_thing, ...)` as opposed to `glue(.envir = list_ish_thing, ...)` in one or more places here in this package.

I started to see if I could make a PR for you, but I couldn't get codebook's dev deps installed:

```
> pak::pak()
✔ Updated metadata database: 3.06 MB in 6 files.                           
ℹ R 4.3 aarch64-apple-darwin20 packages are missing from Bioconductor       
✔ Updating metadata database ... done                                      
Error:                                                                     
! error in pak subprocess
Caused by error: 
! Could not solve package dependencies:
* deps::/Users/jenny/rrr/codebook: Can't install dependency userfriendlyscience
* userfriendlyscience: Can't find package called userfriendlyscience.
```

Backstory in glue:

https://github.com/tidyverse/glue/issues/308
https://github.com/tidyverse/glue/commit/e2b74ff17704261b5a7da25b4ebd2dec94740764

**Comment by jennybc** [comment]
Before I rolled back / postponed this change in glue, here's what I saw in my revdep checks for codebook:

https://github.com/tidyverse/glue/blob/c98f106c420d658da8010627d4a2a4d19f97d758/revdep/problems.md#codebook

**Comment by jennybc** [comment]
I released glue 1.7.0 today without this change and then re-introduced the change. So you can expect it to be present in glue's next release, which I have no concrete plans for. But barring an unforeseen release in the next ~2 weeks, I will consider this issue as me having given plenty of notice of the change.

This also means you can check your package in the presence of dev glue to test your own fix for the issue.

**Comment by jennybc** [comment]
glue 1.8.0 has been accepted on CRAN now.

**Comment by rubenarslan** [comment]
closed in 0.9.5 sorry for the delay
